pkgs/os-specific/linux/tailor-gui/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, rustPlatform
+, cargo
+, rustc
+, pkg-config
+, desktop-file-utils
+, appstream-glib
+, wrapGAppsHook4
+, meson
+, ninja
+, libadwaita
+, gtk4
+}:
+let
+
+  src = fetchFromGitHub {
+    owner = "AaronErhardt";
+    repo = "tuxedo-rs";
+    rev = "f356f05d8fe40de3c5121c11a5a9b37047501440";
+    hash = "sha256-RVauCQn8T6q+c7z0nYGtxdV/89IWAGSsqHL2xm0OTgI=";
+  };
+
+  sourceRoot = "source/tailor_gui";
+
+  name = "tailor_gui";
+
+  # NOTE: OfBorg doesn't like importTOML from a subdirectory
+  version = "0.2.1";
```

Comment:
That would be IFD

Reply:
Thanks for the info.
Shall I remove the comment?

Reply:
Yeah